infra/src/database_cms.tf

```diff
@@ -24,9 +24,9 @@ module "cosmosdb_account" {
   enable_automatic_failover        = true
 
   additional_geo_locations = [{
-    location          = "northeurope"
+    location          = "italynorth"
```

Comment:
Will it be possible to change the name of the database in italynorth?  I mean, currently the name of CosmosDB is `io-p-cosmos-services-cms`, but in italynorth we would like to have a CosmosDB resource named `io-p-itn-svc-cosno-01`, is it possible?

Reply:
on azure portal the replica has already been replaced, there is no more `northeurope` now there is `italynorth`, with this PR I aligned the code, the only new thing is the private endpoint, about the name of the DB now is the same as before

Reply:
Okay, I got it. Shall we, at least, apply the “new naming conventions” to the private endpoint?
I mean, from this
```
resource "azurerm_private_endpoint" "cosmos_db" {
  name                = "${local.project}-itn-cosmos-services-cms"
  location            = "italynorth"
  resource_group_name = azurerm_resource_group.rg.name
  subnet_id           = data.azurerm_subnet.private_endpoints_subnet_itn.id

  private_service_connection {
    name                           = "${local.project}-itn-cosmos-services-cms-private-endpoint"
    private_connection_resource_id = module.cosmosdb_account.id
    is_manual_connection           = false
    subresource_names              = ["Sql"]
  }
}
```
to this
```
resource "azurerm_private_endpoint" "cosmos_db" {
  name                = "${local.project}-itn-svc-cosno-pep-01"
  location            = "italynorth"
  resource_group_name = azurerm_resource_group.rg.name
  subnet_id           = data.azurerm_subnet.private_endpoints_subnet_itn.id

  private_service_connection {
    name                           = "${local.project}-itn-svc-cosno-pep-01" #FIXME: how to name this?
    private_connection_resource_id = module.cosmosdb_account.id
    is_manual_connection           = false
    subresource_names              = ["Sql"]
  }
}
```
Or do you think it might be more confusing?

Reply:
no that's good, i've changed it